middlewares/auth.middleware.js

```diff
@@ -0,0 +1,9 @@
+const isAuth = (req, res, next) => {
+  if (req.isAuthenticated()) {
+      next();
+  } else {
+      res.status(401).json({ msg: 'You are not authorized to view this resource' });
```

Comment:
Not code-related, but for consistency with other user messages, should this have a period after "resource"?